mendix/README.md

```diff
@@ -0,0 +1,39 @@
+# Mendix
+
+## Overview
+
+[Mendix][1] is a market [leading low-code IDE][2] for professional developers. A powerful visual model-driven development environment to build apps on the Mendix Platform.
+
+With Mendix, you can easily create, change, integrate, test and deploy your applications. Moreover, you can manage branch lines and security as well as extend your app with custom code using the built-in editors.
+
+In addition to the native monitoring capabilities out of the box, Mendix also supports integrations to leading 3rd party application performance management vendors such as DataDog. Refer to our documentation to see all our [supported integrations][3].
+
+The Mendix integration with DataDog enables you to consume and monitor your Mendix ecosystem. The [DataDog integration][4] from Mendix will enable you to monitor
+Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, SaaS(Software as a Service) environment metrics and any custom metrics you choose to
+configure in Mendix Studio Pro.
+
+
+> **Note**:  This integration is applicable to Mendix Cloud & Mendix Cloud Dedicated deployment models.
+
+
+## Setup
+
+To enable the DataDog integration for Mendix applications running on Mendix cloud follow the instructions [here][6].
+
+
+## Data Collected
+
+The list of metrics available for monitoring once the integration is enabled is detailed in the [Mendix documentation][7].
+
+
+## Support
+
+Need help? Contact [Mendix support][5].
+
+[1]: https://mendix.com/
+[2]: https://www.mendix.com/blog/a-low-code-leader-composing-the-modern-enterprise-with-mendix/
+[3]: https://docs.mendix.com/developerportal/operate/monitoring-with-apm/
```

Comment:
```suggestion
```

Reply:
@EH - Is there a reason to remove this link? We would prefer to keep it as it provides an overview of how the integration works with Mendix overall

Reply:
cc - @pappachino

Reply:
@eho1307 We would like to keep the referenced link as it provides a generic starting point for all APMs including Datadog. I've updated the README accordingly.